lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -295,10 +295,10 @@ def self.available_options
                                   optional: true,
                                       type: String,
                               verify_block: proc do |value|
-                                accepted_formats = ["mapping.txt"]
+                                accepted_formats = [".txt"]
                                 if value
                                   UI.user_error!("Couldn't find mapping.txt at path '#{value}'") unless File.exist?(value)
-                                  UI.user_error!("Only \"mapping.txt\" file name is allowed, you provided \"#{File.name(value)}\"") unless accepted_formats.include? File.basename(value)
+                                  UI.user_error!("Only \"mapping.txt\" file name is allowed, you provided \"#{File.name(value)}\"") unless accepted_formats.include? File.extname(value)
```

Comment:
If we're only restricting to `.txt`, should we update the language in the error messages to reflect that?

Reply:
Yep, I missed that, thank you